app/lib/providers/navigation_provider.dart

```diff
@@ -10,7 +10,9 @@ class NavigationProvider extends ChangeNotifier {
     OngoingTravel(),
     TravelHistory(),
   ];
+
```

Comment:
Interesting that `flutter format` didn't do this by my IDE, with the same settings, did